.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: scala
 jdk:
 - oraclejdk8
-script: sbt +test
+script: sbt +test
```

Comment:
Revert this file, this change is not needed as nothing really has changed, you have to exclude this file from Git commit.